### DISCUSSION TITLE
Let's create multiple worlds of increasing difficulty

### DISCUSSION THREAD

**Discussion by sjev** [FieldRobotEvent/Virtual_Field_Robot_Event#5] [Ideas]
Hi everyone,

I taken a quick look at the pull request by Kamaro, it't great!  I've seen that the simple 'stick grid' from the first version has been removed.
I would like to propose that we keep the simplified versions as these will help the teams develop their algorithms in steps of increasing difficulty.  We could just move the 'extra' worlds to a separate folder.
I personally like the 'stick' world as it is a 'hello world' example of row following simulation.
So, please make a folder like `test_worlds` where different envrionments can be kept. No need to throw valuable things away imo.

**Comment by Darkproduct** [comment]
This sounds great. We'll implement a simple example, but with better models. There will be a simple straight row version available. We'll edit the pull request and add some sample worlds.

**Comment by Darkproduct** [comment]
Here is an overview for the new [sample worlds](https://github.com/FieldRobotEvent/Virtual_Field_Robot_Event/blob/master/virtual_maize_field/README.md#sample-worlds). We'll add more if there are new features to test.
